public/datasets/pixar_toy.json

```diff
@@ -0,0 +1,48 @@
+{
+  "card": {
+    "description": "Survey results from a quiz that determines which Pixar character you are based on several quiz questions",
+    "source": "Code.org",
```

Comment:
Are we allowed to reference trademarked characters like this? I think the answer is yes because it's more like referencing a musical group from the Spotify dataset than like a brand-heavy Hour of Code partnerships (Disney for Frozen etc). Just wanted to double check.

Reply:
same for Pokemon

Reply:
I have the same question - I made the Pixar dataset before realizing this potential issue. I plan to bring it up at our LT meeting since I'm not sure who to ask about it. The good news is: I have 5 other similar datasets that don't reference pop culture, so I can use those if needed =)